client/src/pages/Initial/LandingPageUzma.jsx

```diff
@@ -0,0 +1,36 @@
+import './LandingPageUzma.scss';
+import MainFroshLogo from '../../assets/logo/frosh-main-logo-with-bg.svg';
+import InstagramLogo from '../../assets/social/instagram_icon.png';
+
+const LandingPageUzma = () => {
+  return (
+    <div className="landing-page-uzma-header">
+      <div className="landing-page-uzma-header-text">
+        <img className="landing-page-uzma-icon" src={MainFroshLogo} alt="frosh logo"></img>
+        <h2>Stay tuned...</h2>
+        <h3>
+          The F!rosh team cannot wait to share this with you! Meanwhile, follow us on Instagram!
+        </h3>
+
+        <a href="https://www.instagram.com/froshweek/" target="_blank" rel="noreferrer">
+          <img
+            src={InstagramLogo}
+            className="landing-page-uzma-instagram-logo"
+            alt="Instagram logo links to F!rosh Instagram page"
+          />
+        </a>
+        <p>F!rosh Week</p>
+        <a href="https://www.instagram.com/froshnomore/" target="_blank" rel="noreferrer">
```

Comment:
we have a class within the `App.scss` that removes the blue underline from the link tags called `no-link-style`.
![image](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/86009011/13f82fee-3b0b-459f-8063-3f2a6112bcec)
you can use it like `className="no-link-style"` inside the `<a>` tags if you'd like.
if it's intention you can disregard the comment!